### PR TITLE
Ping command and Oceans fix

### PR DESCRIPTION
**Pull request by jfoulroberts**
- Created a simple ping command to check bot responsiveness.
- Modify oceans command to deny API call if server is in update status.

### CODE REVIEW

File: bot/commands/Commands.py [hunk 1]
```diff
@@ -37,13 +37,17 @@ def __init__(self):
...
 
             for i, k in sorted(oceans.items()):
                 output += "%s: %s\n" % (i, k)
                 total += k
 
-            output += BotLocalizer.OCEANS_TOTAL % total
+            if s.get('status', 0) == 3:
+                output = 'The Legend of Pirates Online is currently closed for an update. Ocean data is unavailable. Try again when the servers are back up!'
+            else:
+			    output += BotLocalizer.OCEANS_TOTAL % total
```
Comment (mfwass) [inline bot/commands/Commands.py]: May you please fix this line.  We require spaces, not tabs.  It's also indented too far.

File: bot/commands/Commands.py [hunk 2]
```diff
@@ -122,4 +126,15 @@ def __init__(self):
...
+        @self.bot.command()
+        async def ping(*args):
+            """
+            Returns a simple message to check if the bot is operating properly.
+            """
+
+            output = 'Pong!'
+            
+            # Response
             await self.bot.say(output)
```
Comment (mfwass) [inline bot/commands/Commands.py]: May you add a newline here?

**Review by mfwass — CHANGES_REQUESTED** [review]
Changes requested are in the above comments.

File: bot/commands/Commands.py (hunk 2 above)
Comment (jfoulroberts) [reply bot/commands/Commands.py]: Added in newest commit.

**Review by mfwass — APPROVED** [review]

**Pull request merged**